docs/source/posts/2023/2023-06-26-week-4-joaodellagli.rst

```diff
@@ -0,0 +1,60 @@
+Week 4: Nothing is Ever Lost
+============================
+
+.. post:: June 26, 2023
+   :author: João Victor Dell Agli Floriano
+   :tags: google
+   :category: gsoc
+
+
+Welcome again to another weekly blogpost! Today, let's talk about the importance of guidance throughout a project.
+
+This Last Week's Effort
```

Comment:
This sentence seem a bit odd to me.
You can either use `This Week's Effort` or `Last Week's Effort`.